rio_tiler/colormap.py

```diff
@@ -94,6 +94,9 @@ def apply_cmap(
     if data.shape[0] > 1:
         raise InvalidFormat("Source data must be 1 band")
 
+    if len(colormap) > 256 or max(colormap) > 256:
```

Comment:
Why do you only check for `> 256` and not `!= 256`?

Reply:
@kylebarron because `make_lut` should work for cmap that have less than 256 value or a max smaller than 256 
https://github.com/cogeotiff/rio-tiler/pull/325/files#diff-1c0d3ee98487c3ad9e96a708cf2b17d8dba80980c2b0bd6faaca3688543bfd41R66-R68

just realizing that it should be `max(colormap) >= 256` 🤦

Reply:
updated the code with comments: https://github.com/cogeotiff/rio-tiler/blob/master/rio_tiler/colormap.py#L97-L101